frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeBranchToParentAction.java

```diff
@@ -53,6 +54,17 @@ public List<SyncToParentStatus> getEligibleStatuses() {
   protected void onUpdate(AnActionEvent anActionEvent) {
     super.onUpdate(anActionEvent);
     syncToParentStatusDependentActionUpdate(anActionEvent);
+    val presentation = anActionEvent.getPresentation();
```

Comment:
Huh why this new piece of logic is needed? 🤔

Reply:
Was it missing **regardless** of this action name improvements?

Reply:
n/a (I ve modified the wrong class, it was late xd)